.scalafmt.conf

```diff
@@ -0,0 +1,32 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Is it a ASF project?

Reply:
Exactly this file is from the Apache Spark repo, so I think we should preserve the header. Overall, this header is very nice to have because it will also simplify the donation of the project to ASF or making it a part of the Apache Spark.

Reply:
yes, but as long as this is not an ASF project I don't think we can say that it is.

Reply:
Ok. I removed the header.